mysql_example/mysql_example.py

```diff
@@ -276,7 +277,8 @@ def dbWriter(sql, rows) :
 c.execute("CREATE TABLE plural_block "
           "(SELECT block_id, donor_id "
           " FROM blocking_map INNER JOIN plural_key "
-          " USING (block_key))")
+          " USING (block_key)"
+          " GROUP BY block_key, donor_id)")
```

Comment:
Have you found instances where the one donor_id is covered multiple times by the same block_key?